src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const LiElement = tree.querySelectorAll('li');
+
+LiElement.forEach((listItem) => {
+  const ulElement = listItem.querySelector('ul');
+
+  if (ulElement) {
+    const span = document.createElement('span');
+    const text = listItem.firstChild;
+
+    span.textContent = text.textContent;
```

Comment:
Accessing `text.textContent` assumes that `text` is a text node. Ensure that `text` is a text node to avoid potential errors.